pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy

```diff
@@ -106,7 +104,7 @@ public class Converter {
             makeCompilerConfiguration(),
             new CodeSource(src, new Certificate[0]),
             getCompilationClassLoader());
-        cu.addSource(src);
+        cu.addSource(src)
```

Comment:
Oops, was messing around with that at one point and didn't replace the semicolon.